views/eApostilles/uploadFiles.ejs

```diff
@@ -80,24 +80,31 @@
     </p>
 
     <div class="multi-file-upload govuk-!-margin-top-9">
-        <div class="multi-file-upload__form">
-            <form action="/upload-file-handler" method="post" enctype="multipart/form-data" data-testid="upload-form">
+        <div class="multi-file-upload__form js-upload-btn">
+            <form id="file-upload-form" action="/upload-file-handler" method="post" enctype="multipart/form-data" data-testid="upload-form" class="js-upload-form">
                 <div class="multi-file-upload__input-wrapper">
                     <label class="form-label-bold multi-file-upload__label" for="file-upload-1">
                         Upload files
                     </label>
-                    <input class="multi-file-upload__input" data-testid="upload-input" id="file-upload-1" name="documents" type="file" multiple accept="application/pdf"
-
-                    <% if (errors.length > 0) { %> aria-describedby="error-list"
-                            <% } %>
-                    >
+                    <input class="js-multi-file-input multi-file-upload__input" data-testid="upload-input" id="file-upload-1" name="documents" type="file" multiple accept="application/pdf"
+                      <% if (errors.length > 0) { %> aria-describedby="error-list"<% } %>
+                    />
                 </div>
-                <button class="govuk-button govuk-button--secondary">
+                <button type="submit" class="govuk-button govuk-button--secondary js-trigger-progress-bar">
                     Upload
                 </button>
             </form>
         </div>
 
+        <div class="js-progress-bar govuk-!-display-none govuk-!-margin-bottom-4" aria-live="polite">
+            <label class="js-upload-progress-text form-label-bold" for="upload-progress-bar">
+                Uploading...
+            </label>
+            <div class="upload-progress--outline"></div>
+            <div class="js-upload-progress-bar upload-progress--bar" aria-valuenow="0" aria-valuemin="0" aria-valuemax="100"></div>
```

Comment:
are the aria-values actually being updated? you may need to throttle your network speed to see if it's getting updated properly. they need to be updated, since screen readers will rely on this for the progress

Reply:
Fair question. I added [this line](https://github.com/UKForeignOffice/loi-application-service/pull/320/files#diff-153f3ee03b38debd99cd3bb44a5151c3a4c9e42eb73378c8617c8c480d8f51bbR51) to update it with client-side js, I don't know if that means I should take off the `aria-valuenow="0"` attribute in the div element 🤷.

Annoyingly it looks like uploading blocks the main thread which made it difficult to inspect the element or write code in the devTools console. So I wrote a little console.log:
```js
console.log(document.querySelector('.js-upload-progress-bar').getAttribute('aria-valuenow'), 'valuenow');
```

Which printed out the following:
<img width="727" alt="Screenshot 2022-01-06 at 11 05 42" src="https://user-images.githubusercontent.com/1377253/148377823-8db690bd-0c90-4f83-bca7-c1a96351cf5c.png">

I didn't test it on a screen reader or anything so I hope it works 🤞